oauth2/github.go

```diff
@@ -143,7 +144,7 @@ func getOrganizations(client *github.Client, log chronograf.Logger) ([]*github.O
 			PerPage: 10,
 		}
 		// Get the organizations for the current authenticated user.
-		orgs, resp, err := client.Organizations.List("", opt)
+		orgs, resp, err := client.Organizations.List(context.Background(), "", opt)
```

Comment:
This should be context.TODO() until context is wire through via
https://github.com/influxdata/chronograf/blob/0d639d5177b72c12d9470e10ea69ff2dfff45a94/oauth2/oauth2.go#L55